### PR TITLE
Add colour to review outcome displays and do not show "Avg. Score" if it is zero (0)

### PR DESCRIPTION
**Pull request by frjo**
Replaces PR: #3380

### CODE REVIEW

File: hypha/apply/funds/templates/funds/includes/review_sidebar_item.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
             <div>
                 <a href="{% url 'apply:submissions:reviews:review' submission_pk=reviewer.submission.id pk=reviewer.review.id %}">
                     <div class="reviews-sidebar__name">
-                        <span>{{ reviewer }}</span>
+                        <span class="truncate">{{ reviewer }}</span>
```
Comment (theskumar) [inline hypha/apply/funds/templates/funds/includes/review_sidebar_item.html]: If truncation is applied, I would recommend a lightweight method for viewing the full version, such as adding a title= attribute.

See an example below:
Screenshot 2023-09-04 at 05 50 30@2x

**Review by theskumar — APPROVED** [review]

**Comment by fourthletter** [comment]
Could you please show an example of how names are truncated? Thank you.

**Comment by frjo** [comment]
@fourthletter See e.g. https://test-apply.hypha.app/apply/submissions/23/

**Comment by fourthletter** [comment]
I currently do not see the benefit, but would happy to discuss further.

## Current
image

## New PR (proposed change)
image

**Comment by frjo** [comment]
@fourthletter 

What parts of this was it that you did not want implemented?

- [ ] Colouring of review outcomes
- [ ] Hiding "Avg. Score" if it is zero (0)
- [ ] Truncation of reviewers names
- [ ] Sorting reviews by recommendation.

**Comment by fourthletter** [comment]
Great feature!

For Hypha Main - all of these would be helpful for adopters. 
- [x] Colouring of review outcomes
- [x] Hiding "Avg. Score" if it is zero (0)
- [x] Truncation of reviewers names
- [x] Sorting reviews by recommendation.

For OTF, only Colouring of review outcomes. Could we remove "Avg. Score" completely for OTF? Thanks.

**Comment by frjo** [comment]
OTF runs stock Hypha. So what goes in to Hypha is what OTF will use.

ARDC have a custom version of Hypha and are having constant issues with keeping it updated from Hypha stock. I do not want to waist time on that for OTF.

**Comment by frjo** [comment]
I have removed truncation and sorting from this PR now.

**Pull request merged**